### DISCUSSION TITLE
Github hacked or my account?

### DISCUSSION THREAD

**Discussion by petrowsky** [community/community#118438] [Code Security] [answered]
### Select Topic Area

Question

### Body

I just got an email today about a commit which is over 14 years old and new images were added to it from githubusercontent.com.

My account is secured with 2FA using a secure key. I don't think my account is hacked. Is github having issues? The commit is here and the images of MetaNova are the injected new content.

https://github.com/petrowsky/filemaker.tmbundle/commit/85031fffe48d6de8a82f1d3395060a2e2daba629

![github_hack](https://github.com/community/community/assets/96553/32f1e68c-2011-4cef-a066-81a97e227a92)

**Comment by petrowsky** [comment]
I suspect this is a github.com issue as the account was created 12 hours ago. So somewhere, someone has access to modify repositories.

![github_hack2](https://github.com/community/community/assets/96553/47aa24ed-58fa-4c6a-b1c8-49bdeff5dec6)

**Comment by ettaboyle** [comment, marked as answer]
Hi @petrowsky, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
